packages/components/pie-checkbox-group/src/index.ts

```diff
@@ -85,8 +90,9 @@ export class PieCheckboxGroup extends FormControlMixin(RtlMixin(LitElement)) imp
                 ?disabled=${disabled}
                 aria-describedby="${ifDefined(assistiveText ? assistiveTextId : undefined)}"
                 data-test-id="pie-checkbox-group"
+                class="c-checkbox-group ${isInline && '__inline'}"
```

Comment:
I think `c-checkbox-group--inline` might be more in line (pardon the pun) with BEM

Reply:
Yeah, I thought about it and it felt that with shadow dom encapsulation and styling being bound to each component we can in general use shorter css class names. It is so much easier to read and understand what the classname does when I see just `inline` for example then `c-checkbox-group--inline`. I think currently we have a mix of names in the library. Do we want to strictly follow BEM?

Reply:
I think encapsulation is a good enough argument for me, I'm happy for it to stay the same

Reply:
If we want to change the way we write modifiers, we should probably do this across all components, otherwise, we'll end up with a lot of inconsistency across components.

To match current naming, the classes would be `c-checkboxGroup` and `c-checkboxGroup--inline`.

Happy to chat about if we want to change naming within shadow DOM components.